pom.xml

```diff
@@ -1103,12 +1103,12 @@
             </dependency>
             <dependency>
                 <groupId>${james.groupId}</groupId>
-                <artifactId>blob-export-file</artifactId>
+                <artifactId>blob-export-guice</artifactId>
                 <version>${project.version}</version>
             </dependency>
             <dependency>
                 <groupId>${james.groupId}</groupId>
-                <artifactId>blob-export-file-guice</artifactId>
+                <artifactId>blob-export-file</artifactId>
```

Comment:
Wrong order between the two modules

Reply:
I think I did it right, `blob-export-guice` then `blob-export-file` then `blob-memory`, right?

Reply:
What's before the other in the alphabet, `f` or `g`? :)

Reply:
g then l then m. I do google alphabet table most of time I do ordering in poms :)

Reply:
`blob-export-file` should go before `bolb-export-file`

Reply:
sorry, my miss-reading